src/test/java/io/fabric8/maven/XMLFormatTest.java

```diff
@@ -0,0 +1,29 @@
+package io.fabric8.maven;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.io.StringReader;
+
+import org.approvaltests.Approvals;
+import org.junit.jupiter.api.DisplayNameGeneration;
+import org.junit.jupiter.api.DisplayNameGenerator;
+import org.junit.jupiter.api.Test;
+
+@DisplayNameGeneration(DisplayNameGenerator.ReplaceUnderscores.class)
+class XMLFormatTest {
+
+    @Test
+    public void should_ident() {
+        XMLFormat xmlFormat = XMLFormat.builder().indent(4).build();
+        String result = xmlFormat.format(new StringReader("<root><child/></root>"));
+        Approvals.verify(result);
```

Comment:
:+1: I'm glad you adopted approval testing :-)

Reply:
It's fantastic, thanks for introducing me to it 😀